docs/enhancements/api-for-ignoring-validations.md

```diff
@@ -0,0 +1,132 @@
+---
+title: api-for-skipping-validations
```

Comment:
really a nit, but choose either "ignoring" (in the file name and other places) "bypass" or "skipping" based on the more precise term. currently I see the 3 terms reoccurring in some places

maybe "ignore" is better because "skip" might mean something for not happening in the first place